.travis.yml

```diff
@@ -26,13 +26,13 @@ before_script:
   - psql -U postgres -c 'create database twine_api_test'
   - psql -U postgres -c 'create extension postgis'
 script:
-  - for x in "api" "dashboard-app" "visitor-app" "lib/twine-util"; do cd $x; npm run lint; cd ..; done;
+  - for x in "api" "dashboard-app" "visitor-app" "nativeApp" "lib/twine-util"; do cd $x; npm run lint; cd ..; done;
```

Comment:
can we follow the same naming convention for top level directories?

Reply:
expo forces you to create camel case names for apps when initialising. I didn't search too deeply, but don't feel comfortable changing post creation incase it breaks something internally